scripts/mea/et/internal_witness.py

```diff
@@ -307,15 +311,26 @@ def _parse_model_comments(self):
 
                     # Try match EMG comment
                     # Expect comment like /* TYPE Instance Text */
+
+                    match2 = new_emg_comment.search(text)
+                    if match2:
+                        data = json.loads(match2.group(1))
+                        self._add_emg_comment(file_id, line, data)
+                        self._env_models[self.resolve_function_id(data["name"])].append(data)
```

Comment:
Current format of `_env_models` is: `func_id -> comment`. You need to either keep it or change it everywhere with corresponding comments